draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -580,6 +580,81 @@ To ensure readability and replicability, the examples in this specification do
 not contain decoy digests unless explicitly stated. For an example
 with decoy digests, see (#example-simple_structured).
 
+### Recursive Disclosures {#recursive_disclosures}
+
+The algorithms above are compatible with "recursive disclosures", in which one
+selectively disclosed field reveals the existence of more selectively
+disclosable fields.  For example, consider the following JSON structure:
+
+```json
+{
+    "family_name": "Möbius",
+    "nationalities": ["DE", "FR", "UK"]
+}
+```
+
+When the Holder has multiple nationalities, the issuer may wish to conceal
+presence of any statement regarding nationalities while also allowing the
+holder to reveal each of those nationalities individually.
+This can be accomplished by first making the entries within the `nationalities`
+array selectively disclosable, and then the whole "nationalities" field.
```

Comment:
> I hope this is not considered extra, this feels nicer for non-native speakers..

sure and just noticed this so also let's be more consistent with ticks vs quotes 

```suggestion
This can be accomplished by first making the entries within the "nationalities"
array selectively disclosable, and then making the whole "nationalities" field
selectively disclosable.
```